backend/models/Meeting.js

```diff
@@ -7,7 +7,7 @@ export default (sequelize, DataTypes) => {
     {
       id: {
         allowNull: false,
-        autoIncrement: true,
+        // autoIncrement: true,
         primaryKey: true,
         type: DataTypes.STRING,
```

Comment:
I missed this in the last review from Jude but I think its because the `type` should be `type: DataTypes.INTEGER`. Then I think that should fix your tests without commenting this line